core/lib/presentation/utils/html_transformer/text/standardize_html_sanitizing_transformers.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:convert';
+import 'package:core/presentation/utils/html_transformer/base/text_transformer.dart';
+import 'package:core/presentation/utils/html_transformer/sanitize_html.dart';
+
+class StandardizeHtmlSanitizingTransformers extends TextTransformer {
+
+  static const List<String> mailAllowedHtmlAttributes = [
+    'style',
+    'public-asset-id',
+    'data-filename',
+    'bgcolor',
+  ];
+
+  static const List<String> mailAllowedHtmlTags = [
+    'font',
+    'u',
+    'mcourbo@linagora.com',
```

Comment:
why we need it?

Reply:
We have some emails that contain this tag. I don't know why it is there, or is it user-created?. So to display it we have to allow it here.

<img width="630" alt="Screenshot 2024-10-23 at 17 56 20" src="https://github.com/user-attachments/assets/f8230040-9a4a-4d48-9e06-19cd08653df7">

<img width="413" alt="Screenshot 2024-10-23 at 17 56 05" src="https://github.com/user-attachments/assets/76eb3621-6bb1-44ba-a474-08c873d0701e">

Reply:
This looks **bad**

Reply:
IMO, we will delete it.